config.yaml

```diff
@@ -14,3 +14,11 @@ options:
     type: string
     default: ''
     description: Secret key
+  mode:
+    type: string
+    default: 'server'
+    description: "Mode of operations. Possible values: server, gateway"
+  gateway_storage_service:
+    type: string
+    default: ''
+    description: "Storage service used by gateway to store objects. Possible values: s3, azure"
```

Comment:
If this is required for `mode=gateway` operation, can we add a comment saying it must be set?